case_studies/sdss_galaxies/plots.py

```diff
@@ -441,7 +441,8 @@ def compute_data(self, frame: Union[SDSSFrame, SimulatedFrame], encoder, decoder
         h_end = ((frame.image.shape[2] - 2 * bp) // 4) * 4 + bp
         w_end = ((frame.image.shape[3] - 2 * bp) // 4) * 4 + bp
         coadd_params: FullCatalog = frame.get_catalog((h, h_end), (w, w_end))
-        coadd_params.plocs += bp
+        # need to adjust by bp since relative to h = w = bp.
+        coadd_params.plocs += bp - 0.5
```

Comment:
I would suggest doing this adjustment within your plotting code rather than mutating the FullCatalog.

Reply:
Thanks!